test/common/router/header_formatter_test.cc

```diff
@@ -298,6 +299,12 @@ TEST(HeaderParserTest, TestParseInternal) {
        {"Invalid header configuration. Un-terminated variable expression 'VAR after'"}},
       {"% ", {}, {"Invalid header configuration. Un-terminated variable expression ' '"}},
 
+      // Mismatched array beginning and termination.
+      {"%START_TIME([\"testing)%",
```

Comment:
I would do this with UPSTREAM_METADATA, but I think it's fine to add the test case.

Reply:
Done